### ISSUE TITLE
I can't get Live TV to play

### ISSUE THREAD

**Issue by jonpmyatt** [jellyfin/jellyfin-tizen#268]
Live TV isn't working for me. I'm using tvheadend and I can confirm that it works fine with Jellyfin media player and in a web browser, so I know tvheadend and the Jellyfin server are working ok. Server is latest 10.9.4. Whenever I try and play live TV I just get a spinner then a black screen and I have to back all the way out of the app. TV model is QE65Q80RATXZG. 
Would appreciate any suggestions :)

**Comment by jonpmyatt** [comment]
Changed the allowed resolution on the client and it's playing now.

**Comment by dmitrylyzo** [comment]
> Changed the allowed resolution on the client and it's playing now.

I guess, it started transcoding. Previously, it was probably trying to direct play.
What `Original Media Info` in the `Playback Info`?

**Comment by jonpmyatt** [comment]
Yeah, it's transcoding. Original Media Info just says: Video Range Type SDR
Elsewhere in the Playback Info it tells me H264, AAC, Bitrate 119.7Mbps 50fps
It would be great if it would direct play...

**Comment by dmitrylyzo** [comment]
> Original Media Info just says: Video Range Type SDR

Under this section:
Container?
Video codec?
Audio codec?

> Elsewhere in the Playback Info it tells me H264, AAC, Bitrate 119.7Mbps 50fps

I guess, this is `Transcoding Info` - transcoding output. We need the original info.

**Comment by jonpmyatt** [comment]
Sorry yes that was transcoding info. Under media info it doesn't give me anything else sorry. Pic attached. 

<img width="669" alt="media-info" src="https://github.com/jellyfin/jellyfin-tizen/assets/52711694/fdfbd2b7-812e-4d49-a5c9-2d2975235ba5">

**Comment by jonpmyatt** [comment]
Re-opened for further investigation

**Comment by kevpen** [comment]
I seem to be having the same issues. 

I can confirm that the steams work on Web and Android clients.

Tizen with Jellyfin 10.9.6 seems to spin. I've tried forcing transcode, and setting max allowed resolution. No errors in client logs.
